yt_dlp/extractor/fuyintv.py

```diff
@@ -0,0 +1,27 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class FuyinTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)fuyin\.(?:tv)/(?:html)/(?P<mov_id>\w+)/(?P<url_id>\d+)(?:\.html)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?fuyin\.tv/html/(?P<mov_id>\w+)/(?P<url_id>\d+)\.html'
```
Why the unnecessary groups?